qiskit_nature/second_q/transformers/active_space_transformer.py

```diff
@@ -359,7 +358,9 @@ def _validate_num_orbitals(self, nelec_inactive: int, particle_number: ParticleN
 
     # TODO: can we efficiently extract this into the base class? At least the logic dealing with
     # recursion is general and we should avoid having to duplicate it.
-    def _transform_property(self, prop: Property) -> Optional[Property]:
+    def _transform_property(
```

Comment:
This will get refactored further, once the `ElectronicStructureProblem` has reached its final form